src/tests/100-bookingTest.js

```diff
@@ -119,3 +128,42 @@ describe('Book an accomadition facility', () => {
       });
   });
 });
+describe('Get User Booking accomadition request', () => {
+  it('should be able to get user booking request', (done) => {
+    chai.request(app).get('/api/v1/trips/booking')
+      .set('token', managerSignin)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
```

Comment:
Please add more.

Reply:
Thank you @NiyoEric, I have already implemented